kontroller/qml/ServerSettingsPage.qml

```diff
@@ -245,9 +242,9 @@ You need to enable HTTP remote access in kodi for this to work.")
             Label {
                 anchors.left: parent.left
                 anchors.right: parent.right
-                text: qsTr("Select here the volume plugin to use. Default (Kodi) is to use kodi integrated \
-volume. If using an amplifier to control the volume, select the corresponding plugin and configure it \
-to make the remote control the amplifier volume.")
+                text: qsTr("Here you can change the default integrated Kodi volume plugin. \
```

Comment:
Should maybe make the distinction between amp input signal volume adjustment and
amplifier attenuation.

Reply:
The new text is misleading. This does not change kodi configuration, but kontroller one. IE in the case you choose the miniDSP plugin, kontroller talks directly to the device, bypassing completely kodi. There are two possibilities:
* using kodi volume (which is the default). In that case, volume control uses kodi API.
* using miniDSP volume. In that case, kontroller talks directly to the miniDSP device, using its own protocol. It does not touch kodi volume. Note that in that case, if kodi volume was muted, for example, you will never hear anything, whatever volume you apply on the device, since it won't receive anything from kodi.

I don't know how to express this briefly and clearly.

Reply:
I avoided the technicalities. The only thing I can think of is a warning when option 2 is selected to be cautious to avoid playback at full volume. Or just starting it at 10% and hope it isn't connected to something with high sensitivity.

Reply:
What do you think of: 
“Here you can change the way you adjust the volume. By default, kodi internal volume is controlled. If you have an external amplifier, you can select and configure the corresponding plugin so that the amplifier volume is controlled by the remote instead” ?